flink-table-store-core/src/main/java/org/apache/flink/table/store/CoreOptions.java

```diff
@@ -695,6 +691,44 @@ public InlineElement getDescription() {
         }
     }
 
+    /**
+     * Validate the {@link TableSchema} and {@link CoreOptions}.
+     *
+     * <p>TODO validate all items in schema and all keys in options.
+     *
+     * @param schema the schema to be validated
+     * @param options the options to be validated
+     */
+    public static void validateTableOptions(TableSchema schema, CoreOptions options) {
```

Comment:
Minor: this options can generated by `schema`. Just `validateTableOptions(TableSchema schema)` is OK.